playbook.yml

```diff
@@ -0,0 +1,73 @@
+- name: Run todo app
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_key
+    prompt: What is your trello key?
+    private: no
+
+  - name: trello_token
+    prompt: What is your trello token?
+    private: no 
+
+  - name: secret_key
+    prompt: What is your secret key?
+    private: no  
+
+  tasks:
+  - name: Install git
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes
+
+  - name: Install python3
+    ansible.builtin.yum:
+      name: python3
+    become: yes
+
+  - name: Install poetry
+    ansible.builtin.shell:
+      cmd: 'curl -sSL https://install.python-poetry.org | python3 -'

```

Comment:
Not super important but you'll probably see that this re-runs each time, then skips installing when it sees it's already done. Adding the below can tell it to skip if poetry already exists:
```
      become: yes
      args:
        creates: ~/.local/bin/poetry
```